src/v2/coins/abstractUtxoCoin.js

```diff
@@ -1121,13 +1132,30 @@ class AbstractUtxoCoin extends BaseCoin {
       const baseKeyPath = deriveKeys(deriveKeys(keys, 0), 0);
       const userKeyArray = deriveKeys(baseKeyPath, 0);
       const changeKeyArray = deriveKeys(baseKeyPath, 1);
+      const userKeyArraySW = deriveKeys(baseKeyPath, 10);
```

Comment:
These are not great names, maybe `p2shInternalPublicKeys`, `p2shExternalPublicKeys`, `segwitInternalPublicKeys`, `segwitExternalPublicKeys`? Not really in love with these either, so feel free to come up with something a little more descriptive if you so choose.